### ISSUE TITLE
let the data.frame be data.frame, and data.table be data.table.

### ISSUE THREAD

**Issue by GreenEric** [xiaodaigh/disk.frame#85]
Hi, Xiaodaign,
Thanks for the great work. I noticed that, due to the legacy issue from some other package, the `data.frame` is more pleasible for `data.frame`, than `data.table`.
What if I save/read the data as data.fram, and convert it back to data.table within the r-script, will it be workable?
Thanks for your advice.

**Comment by xiaodaigh** [comment]
I actually don't understand what you are asking here. Please elaborate

**Comment by xiaodaigh** [comment]
I think you want it so that disk.frame doesn't auto convert data.frames to data.table. Is that correct?

**Comment by iqis** [comment]
@GreenEric  I think you could find work around this issue pretty easily, as a `data.table` _is_ a `data.frame`

**Comment by xiaodaigh** [comment]
That is true. It's still nice to be able to preserve that in a future release.

**Comment by xiaodaigh** [comment]
Let the user choose their preferred data.frames which are "base"=plain "data.frame" "data.table" or "tibble", so the returned data.frame is in the format most preferred by the user.
